packages/mobile/src/recipients/reducer.ts

```diff
@@ -44,7 +44,6 @@ export const recipientsReducer = createReducer(initialState, (builder) => {
         ...initialState,
         ...state,
         ...hydrated,
-        phoneRecipientCache: initialState.phoneRecipientCache,
```

Comment:
this was causing the cache to be lost after every app restart which is undesirable for a cache i think...